src/Servant/Elm/Internal/Orphans.hs

```diff
@@ -12,3 +12,8 @@ instance ElmType ElmDatatype where
 
 
 instance ElmType NoContent
+
+-- TODO: Generate Elm functions that can handle the response headers. PRs
+-- welcome!
+instance (ElmType a) => ElmType (Headers ls a) where
```

Comment:
I don't think we want to add this back.

We moved the instance to `elm-export`: https://github.com/NoRedInk/elm-export/commit/4b68744a3af70f75ed7d5cb6782454d87cdcd3d3.

If we add this back, we've no clue which instance ghc will chose when it decides to choose one. We can hope and pray that the two instances line up, but if one of them changes, we're really messed up when ghc decides which instance to choose.

I think we should drop this instance again and do one of the following:
* Depend on our fork of `elm-export`.
    It doesn't look like our changes are going to be merged into upstream anytime soon, so no sense trying to keep parity with it. The tests will start compiling then.
* Remove the test for `Headers`.
    If we can't support this data type, we shouldn't test against it.

No preference for either option, but I don't think re-adding this orphan is the way to go.

Reply:
Aside, this is a prime example of why working with these two dependencies has been frustrating for us. You start out wanting to make a seemingly innocuous change, and it ends up expanding into a whole web of problems.

Reply:
thanks for the explanation and proposed solutions. will address asap.

Reply:
@joneshf I removed the orphan and change the dependency to use our fork.